consensus/core/src/transaction.rs

```diff
@@ -217,6 +203,8 @@ impl TransactionConsumer {
 pub struct TransactionClient {
     sender: Sender<TransactionsGuard>,
     max_transaction_size: u64,
+    max_transactions_in_block_bytes: u64,
+    max_transactions_in_block_size: u64,
```

Comment:
```suggestion
    max_transactions_in_block_count: u64,
```
optional: or use the original config field. `size` may confuse with bytes size.

Reply:
`count` sounds good 👍